### DISCUSSION TITLE
使用百度翻译引擎，我只翻译了一单词，历史记录里也只显示一条，但是百度的翻译平台统计我翻译了473个字符，不知道是啥原因，是自动翻译的吗？如何关比自动翻译功能。

### DISCUSSION THREAD

**Discussion by dupengboo** [YiiGuxing/TranslationPlugin#2488] [Q&A]
使用百度翻译引擎，我只翻译了一单词，历史记录里也只显示一条，但是百度的翻译平台统计我翻译了473个字符，不知道是啥原因，是自动翻译的吗？如何关比自动翻译功能。

**Comment by YiiGuxing** [comment]
在插件的设置页面底部有一个选项：翻译文档，这个选项勾选后，当你查看文档时会自动翻译文档，所以你多出的字符是这个的。

**Reply by dupengboo** [reply]
翻译文档的功能我知道，而且一直是关闭的，有没有其他原因？
如何设置，只在我鼠标选中词汇的时候才翻译？

**Reply by YiiGuxing** [reply]
那就没有其他的了
